rusk-wallet/src/gql.rs

```diff
@@ -229,6 +268,33 @@ impl GraphQL {
         Ok(response)
     }
 
+    /// Query the archival node for moonlight transactions of `address`
+    /// in the given `block`.
+    ///
+    /// # Errors
+    /// This method errors if there was an error while sending the query,
+    /// or if the response body is not in JSON format or encoded correctly.
```

Comment:
I assume that this will also error if the `Address` given is not a public-address.

Reply:
Same as the `moonlight_history` function, too.